packages/scm/src/browser/style/index.css

```diff
@@ -102,7 +102,11 @@
 }
 
 .theia-scm-input-message-container textarea:not(:focus) {
-    border: none;
+    border: var(--theia-border-width) solid var(--theia-input-background);
```

Comment:
background color for border color?

Reply:
> Are you sure that there is a bug at all.

Yes: https://github.com/eclipse-theia/theia/issues/7338#issue-581279560

Reply:
ok, but it should be `--theia-input-border` color 

Please see https://code.visualstudio.com/api/references/theme-color#input-control

> input.border: Input box border.

Reply:
Also you can have a look at reference implementation from VS 
Code: https://github.com/microsoft/vscode/blob/89906c073a8fc8e60f344c5623f5c73adcf039f1/src/vs/workbench/contrib/scm/browser/repositoryPane.ts#L1094

Reply:
> ok, but it should be `--theia-input-border` color

Well, ideally it should be nothing.  The `theia-input` CSS should work out of the box.

Reply:
It belongs already: https://github.com/eclipse-theia/theia/blob/9457f8087bb421b7aa1c6cf124c6dfe57659df5f/packages/core/src/browser/style/index.css#L58

Not sure why we don't use theia-input, probably because scm has some custom logic for notification messages and cannot reuse theia-input. cc @jbicker ?

Reply:
> Not sure why we don't use theia-input

We use it, please read [above](https://github.com/eclipse-theia/theia/pull/7340#issuecomment-599174291).

> The commit message `textArea` has the `theia-input` class (on my branch):

Reply:
ok, makes sense, sounds like we should remove this rule and test

Reply:
Just as an FYI, I took the approach of using the background color by default since not doing so would cause #3544 again, and a similar approach was taken for the siw as well:

https://github.com/eclipse-theia/theia/blob/9457f8087bb421b7aa1c6cf124c6dfe57659df5f/packages/search-in-workspace/src/browser/styles/index.css#L57-L62

Reply:
> ok, makes sense, sounds like we should remove this rule and test

Unfortunately, it looks like `textarea` contain special handling in order to work correctly and removing the rules does not work as intended.

Reply:
> Unfortunately, it looks like `textarea` contain special handling in order to work correctly and removing the rules does not work as intended.

I would be okay with the quick-fix then, especially if `theia-input` class does not work as it is intended to.

Reply:
> Unfortunately, it looks like textarea contain special handling in order to work correctly and removing the rules does not work as intended.

It looks like overlook in the original theming PR well. Please file an issue as well. We should not let such things happen, it will degrade support of theming. In order to test we need a theme which change `input.border` color and then we should compare what happens in Theia and VS Code and then align. 

Fine as a work-around for now.

Reply:
☝️ it is reproducible with HC theme, look at how it looks in VS Code and Theia without focus

Reply:
> But the bug on master as well, please file an issue.

I assume it is handled by the epic https://github.com/eclipse-theia/theia/issues/6743?

Reply:
I've been working on fixing input validation:

<img width="320" alt="Screen Shot 2020-03-16 at 12 36 45 PM" src="https://user-images.githubusercontent.com/40359487/76780218-05913300-6783-11ea-9211-e4adc9de503a.png">

This is with the correct theming applied.

Reply:
@vince-fugnitto It is not about validation without any errors and focus border should be shown in HC theme as in VS Code.

Reply:
> @vince-fugnitto It is not about validation without any errors and focus border should be shown in HC theme as in VS Code. It cannot work if we use background color for border color.

I have a pull-request handling the theming for validation: #7351
Would you like me to handle the `hc` theme as well for inputs? I understand that `hc` theming was omitted as part of the first iteration of theming support #6743

Reply:
HC theme is a way to reproduce the issue, it does not work for any theme which defines different input.border.